tox.ini

```diff
@@ -0,0 +1,17 @@
+[tox]
+envlist = ui-tests, flake8
+skipsdist = True
+
+[testenv:ui-tests]
+passenv = DISPLAY
+deps = -rfrontend/test/ui/requirements/requirements.txt
+commands =
+    pytest -v --driver=Firefox --firefox-path=firefox/firefox frontend/test/ui {posargs}
```

Comment:
`frontend/test/ui` should be the default positional argument, to allow specifying an alternate path: `{posargs:frontend/test/ui}`.